src/flynt/api.py

```diff
@@ -126,22 +150,19 @@ def fstringify_files(
     total_expressions = 0
     start_time = time.time()
     for path in files:
-        (
-            changed,
-            count_expressions,
-            charcount_original,
-            charcount_new,
-        ) = _fstringify_file(
+        result = _fstringify_file(
             path,
             state,
         )
-        if changed:
-            changed_files += 1
-            total_expressions += count_expressions
-        total_charcount_original += charcount_original
-        total_charcount_new += charcount_new
-
-        status = "modified" if count_expressions else "no change"
+        if result:
```

Comment:
what does bool conversion do for a dataclass without __bool__ method?

Reply:
A dataclass is truthy by default, like any plain object.